### PR TITLE
[MODORGSTOR-148] - BE - Implement CRUD for Banking Account Types

### PR DESCRIPTION
**Pull request by imerabishvili**
## Approach
https://issues.folio.org/browse/MODORGSTOR-148

## Pre-Merge Checklist:
Before merging this PR, please go through the following list and take appropriate action.

- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code are 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added, or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all the appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally, all the PRs involved in breaking changes would be merged on the same day
to avoid breaking the folio-testing environment.
Communication is paramount if that is to be achieved,
especially as the number of inter-module and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems,
ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

**Comment by Abdulkhakimov** [comment]
Please update the module descriptor

**Comment by imerabishvili** [comment]
> Please update the module descriptor

File: src/main/resources/templates/db_scripts/schema.json [hunk 1]
```diff
@@ -164,6 +164,17 @@
           "tOps" : "ADD"
         }
       ]
+    },
+    {
+      "tableName": "banking_account_types",
+      "fromModuleVersion": "mod-organizations-storage-4.6.0",
```
Comment (SerhiiNosko) [inline src/main/resources/templates/db_scripts/schema.json:170]: 4.6.0 already released, you should use next release version - 4.7.0

File: src/test/java/org/folio/rest/impl/BankingAccountTypesTest.java [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+        SAMPLE_BANKING_ACCOUNT_TYPE_ID);
+  }
+
+  @AfterAll
+  static void afterAll() {
+    purge(TENANT_HEADER);
+    prepareTenant(TENANT_HEADER, false, false);
+  }
+
+  private Future<Void> runSQLTx(String sqlString, boolean startFirst, String failureMessage) {
```
Comment (SerhiiNosko) [inline src/test/java/org/folio/rest/impl/BankingAccountTypesTest.java:70]: common methods can be extracted into base TestBase class

Comment (imerabishvili) [reply src/test/java/org/folio/rest/impl/BankingAccountTypesTest.java]: done

**Review by SerhiiNosko — APPROVED** [review]

**Review by Abdulkhakimov — APPROVED** [review]

**Pull request merged**